server/pulp/server/webservices/views/tasks.py

```diff
@@ -67,6 +74,32 @@ def get(self, request):
         serialized_task_statuses = [task_serializer(task) for task in raw_tasks]
         return generate_json_response_with_pulp_encoder(serialized_task_statuses)
 
+    @auth_required(authorization.DELETE)
+    def delete(self, request):
+        """
+        Delete the tasks with status as finished, error, timed-out and skipped
+
+        :param request: WSGI request object
+        :type  request: django.core.handlers.wsgi.WSGIRequest
+
+        :return: Response containing None or pulp Exception
+        :rtype:  django.http.HttpResponse or pulp.Exception
+        """
+        task_state = request.GET.getlist('state')
+        if not task_state:
+            raise pulp_exceptions.PulpCodedForbiddenException(
```

Comment:
I thought it to be 403 because giving no arguments is forbidden. Client can't keep the argument list empty in case of delete. However, http 400 also makes sense because server won't understand the client's request.